pkg/status/templates/header.tmpl

```diff
@@ -86,6 +86,15 @@ NOTE: Changes made to this template should be reflected on the following templat
     {{ $key }}: {{ $value }}
   {{- end }}
 
+{{- if eq .config.fips_enabled "true" }}
+
+  FIPS proxy
+  ==========
+    FIPS proxy is enabled. All communication will be routed to a local FIPS proxy:
+      - Local address: {{ .config.fips_local_address }}
+      - Starting port range: {{ .config.fips_port_range_start }}
```

Comment:
```suggestion
      - Starting port: {{ .config.fips_port_range_start }}
```
..or should we actually have a count and give the full range (such as "3833-3841")?

Reply:
A user don't setup each port between the agent and the proxy. Setting the same starting port between both is enough. I don't think we need to display the full list. Also we might add port in the future for new product and that would have no impact on the configuration.

Overall I think we should display information that matter in the status page.

Reply:
I think it'd be ideal for the status page here to show the full port range that is used (it does give the user visibility into all the ports that are used for this).

If it's not trivial to implement then I think the starting port is fine and this should read `Starting port` instead of `Starting port range`.